docs/apache-airflow/installation/setting-up-the-database.rst

```diff
@@ -22,7 +22,7 @@ Airflow requires a database. If you're just experimenting and learning Airflow,
 default SQLite option. If you don't want to use SQLite, then take a look at
 :doc:`/howto/set-up-database` to setup a different database.
 
-Usually, you need to run ``airflow db upgrade`` in order to create the database schema that Airflow can use.
+Usually, you need to run ``airflow db init`` in order to create the database schema that Airflow can use.
```

Comment:
Well. This is not entirely true. 

* `airflow db upgrade` creates DB schema 
* `airflow db init` also does what upgrade does but it also adds default connections for example - which might be not expected. 

I'd say we should describe both usages here.

Reply:
Agree with Jarek